### PR TITLE
FlexBox added

### PR DESCRIPTION
**Pull request by kabirdeewan**

### CODE REVIEW

File: CSSFlexbox/flexshrink.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<style>
```
Comment (kaustubhdeokar) [inline CSSFlexbox/flexshrink.html]: Add doctype, header, proper identation.

File: CSSFlexbox/flexwrap.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<style>
```
Comment (kaustubhdeokar) [inline CSSFlexbox/flexwrap.html]: Add doctype, header, proper identation.

File: CSSFlexbox/tweet.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
+<style>
```
Comment (kaustubhdeokar) [inline CSSFlexbox/tweet.html]: Add doctype, header, proper identation.

**Review by kaustubhdeokar — CHANGES_REQUESTED** [review]

File: CSSFlexbox/flexshrink.html (hunk 1 above)
Comment (kabirdeewan) [reply CSSFlexbox/flexshrink.html]: On it.

**Pull request merged**